plugin.cpp

```diff
@@ -144,9 +144,9 @@ int slurm_spank_local_user_init(spank_t sp, int ac, char **av) {
   if (!args.file) {
     // ensure that --uenv-mount is not present, getopt is called here
     // because it is not allowed to be called in `spank_init`.
-    char** tmp{nullptr};
-    spank_err_t ret = spank_option_getopt(sp, &mount_point_arg, tmp);
-    if (ret != ESPANK_ERROR) {
+    char* buf;
+    spank_err_t ret = spank_option_getopt(sp, &mount_point_arg, &buf);
```

Comment:
Would this be better if it was made in `slurm_spank_init_post_opt`? I am assuming that the check is made here to test locally for an invalid argument, instead of checking on the compute nodes.

`slurm_spack_local_user_init` is called after the allocation is made, which would make the user wait until after the allocation has been made, and would also affect other jobs by requiring an allocation.

Reply:
Calling `spank_option_getopt` isn't allowed in `init_post_opt`:
```
slurm_spank_init_post_opt: Item not available from this callback
```